bitcoin/src/psbt/mod.rs

```diff
@@ -767,8 +767,10 @@ pub enum SignError {
     NotEcdsa,
     /// The `scriptPubkey` is not a P2WPKH script.
     NotWpkh,
-    /// Sighash computation error.
-    SighashComputation(sighash::Error),
+    /// Sighash computation error (segwit v0 input).
+    SegwitV0Sighash(sighash::SegwitV0Error),
```

Comment:
This variant looks redundant since it's contained in the other one? Though this gives information that the operation didn't involve P2WPKH...

Reply:
Fair point, it originates in a different error path. But more to the point, I'm not fused about `psbt` stuff here. I'll keep it in mind over in `tcharding/rust-psbt` though when I upgrade to this version of `rust-bitcoin`.